crates/rattler_package_streaming/tests/extract.rs

```diff
@@ -111,7 +111,7 @@ fn test_extract_conda(#[case] input: Url, #[case] sha256: &str, #[case] md5: &st
     println!("Target dir: {}", temp_dir.display());
     let file_path = tools::download_and_cache_file(input, sha256).unwrap();
     let target_dir = temp_dir.join(file_path.file_stem().unwrap());
-    let result = extract_conda(
+    let result = extract_conda_via_streaming(
```

Comment:
can we keep here `extract_conda ? I think it will still call underneath `extract_conda_via_streaming` so we will be sure that we didn't change old implementation somehow

Reply:
same response as in https://github.com/conda-incubator/rattler/pull/797#discussion_r1698415712. This test was testing the innermost method  that I just renamed to `extract_conda_via_streaming`. I'm happy to rename it back to `extract_conda` to get rid of this confusing state if it helps!

Reply:
ah, thanks for clarifying this!